project-keeper/src/main/resources/templates/.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Generate sha256sum files
         run: |
           cd target
-          find . -maxdepth 1 -name *.jar -exec bash -c 'sha256sum {} > {}.sha256' \;
+          find . -maxdepth 1 -name \*.jar -exec bash -c 'sha256sum {} > {}.sha256' \;
```

Comment:
:+1: 

I think we may have to do it in other projects as well...

Reply:
PK will take care of this automatically.
This has the disadvantage, that PK also broke it automatically 😃